assets/js/atomic/blocks/product-elements/button/types.ts

```diff
@@ -0,0 +1,37 @@
+export interface BlockAttributes {
```

Comment:
nitpick: move this down so all exported interfaces are together, at the bottom of the file. It reads more naturally that way